src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ComboBox.cs

```diff
@@ -72,6 +78,52 @@ public void Init()
 
 		const int BorderThicknessAdjustment = 2; // Deduct BorderThickness on PopupBorder
 
+		[TestMethod]
+		public async Task When_IsEditable_False()
+		{
+			// EditableText is only available in fluent style.
+			using var _ = StyleHelper.UseFluentStyles();
+			var SUT = new ComboBox();
+			await UITestHelper.Load(SUT);
+
+			Assert.IsFalse(SUT.IsEditable);
+			Assert.IsNull(_editableTextField.GetValue(SUT));
+		}
+
+		[TestMethod]
+		public async Task When_IsEditable_True()
+		{
+			// EditableText is only available in fluent style.
+			using var _ = StyleHelper.UseFluentStyles();
+			var SUT = new ComboBox() { IsEditable = true };
+			await UITestHelper.Load(SUT);
+
+			Assert.IsTrue(SUT.IsEditable);
+			Assert.IsNotNull(_editableTextField.GetValue(SUT));
+		}
+
+		[TestMethod]
+		public async Task When_IsEditable_False_Changes_To_True()
+		{
+			if (!ApiInformation.IsPropertyPresent(typeof(ComboBox), "IsEditable"))
+			{
+				Assert.Inconclusive();
+			}
+
+			// EditableText is only available in fluent style.
+			using var _ = StyleHelper.UseFluentStyles();
+			var SUT = new ComboBox();
+			await UITestHelper.Load(SUT);
+
+			Assert.IsFalse(SUT.IsEditable);
+			Assert.IsNull(_editableTextField.GetValue(SUT));
+
+			SUT.IsEditable = true;
```

Comment:
I'd tend to think that this particular test is not going to work, as `_editableText` is not reevaluated to make the ComboBox work in that mode. IsEditable dose not seem to be a mode we're supporting properly at this time. I'd say that we can keep this test, but keep it ignored for now.

Reply:
@jeromelaban This test already has:

```csharp
			if (!ApiInformation.IsPropertyPresent(typeof(ComboBox), "IsEditable"))
			{
				Assert.Inconclusive();
			}
```

The intent is that it's now ignored, but once IsEditable is implemented, the test gets  "enabled" automatically.

Reply:
I missed that! Thanks!!